Source/DafnyRuntime/DafnyRuntime.py

```diff
@@ -17,13 +17,24 @@ def __get__(self, instance, owner):
 def print(value):
     builtins.print(string_of(value), end="")
 
+# Dafny strings are currently sequences of UTF-16 code units.
+# To make a best effort attempt at printing the right characters we attempt to decode,
+# but have to allow for invalid sequences.
+def string_from_utf_16(utf_16_code_units):
+    return b''.join([ord(c).to_bytes(2, 'little') for c in utf_16_code_units]).decode("utf-16", errors = 'replace')
```

Comment:
Two small things:
```suggestion
    return b''.join(ord(c).to_bytes(2, 'little') for c in utf_16_code_units).decode("utf-16-le", errors = 'replace')
```

Reply:
Agreed on `-le`, but I don't think we need the `[...]`. As is, `ord(c).to_bytes(2, 'little') for c in utf_16_code_units` is a generator instead of a list, and `bytes.join` only needs an `Iterable`.

Reply:
I think you misread my suggestion and we're in violent agreement ^^ The `[...]` was in your code, and my suggestion was to remove it :)